src/sidebar/activity/tree/ActivityTreeDataProvider.ts

```diff
@@ -32,8 +36,12 @@ class ActivityTreeDataProvider extends TreeDataProvider {
             }
             slots[slot].push(activity);
         });
-        const slotTreeItems = Object.keys(slots).map(key => new ActivitySlotTreeItem(key as DateSlot, slots[key]));
-        return slotTreeItems;
+
+        roots.push(...Object.keys(slots).map(key => new ActivitySlotTreeItem(key as DateSlot, slots[key])));
```

Comment:
```suggestion
        roots.concat(Object.keys(slots).map(key => new ActivitySlotTreeItem(key as DateSlot, slots[key])));
```

Reply:
I would have to reassign your suggestion to roots. So I would like it to remain this way.

`roots = roots.concat(Object.keys(slots).map(key => new ActivitySlotTreeItem(key as DateSlot, slots[key])));`

 Am I wrong?